meilisearch-http/tests/common.rs

```diff
@@ -88,40 +88,6 @@ impl Server {
                 "wordsPosition",
                 "exactness",
             ],
-            "searchableAttributes": [
-                "balance",
-                "picture",
-                "age",
-                "color",
-                "name",
-                "gender",
-                "email",
-                "phone",
-                "address",
-                "about",
-                "registered",
-                "latitude",
-                "longitude",
-                "tags",
-            ],
-            "displayedAttributes": [
-                "id",
-                "isActive",
-                "balance",
-                "picture",
-                "age",
-                "color",
-                "name",
-                "gender",
-                "email",
-                "phone",
-                "address",
-                "about",
-                "registered",
-                "latitude",
-                "longitude",
-                "tags",
-            ],
```

Comment:
Ok so now, default displayed and searchable fields are not shown when there equal to the default value?

Reply:
no, here, the attributes were added at the server creation, but this is not useful now, since the default value in `*`. This is also what hid the bug we add regarding the order documents/settings were added to the engine.